### DISCUSSION TITLE
Morphing wing optimization

### DISCUSSION THREAD

**Discussion by agusbuffa** [jxjo/Xoptfoil2#4] [Ideas]
Hallo,

I'm working on a morphing wing project and I came up with your projects (Xoptfoil-JX and Xoptfoil2). I wanted to ask if it is possible to optimize an airfoil just in the trailing edge zone, it would be a camber optimzation, while the rest of geometry parameters remain like the seed airfoil. If not, I share this idea to be added on this new project.

I would like to get in contact, I have a lot of questions regarding airfoil optimization and using the code hahaha.

**Comment by jxjo** [comment]
Hello,
... could you describe a little bit more in detail what you want to do and what you want to achieve? 
(limiting airfoil shaping to a certain x-range could made possible - but if it's more like setting a flap, than it's getting difficult)

If you have further questions, ask them here (so others can benefit) - or  send me an email to [jochen@jxjo.de](mailto:jochen@jxjo.de)

regards

Jochen
